### DISCUSSION TITLE
[FEATURE] Add CVMFS to be available from notebook?

### DISCUSSION THREAD

**Discussion by oshadura** [CoffeaTeam/coffea-casa#144] [Ideas]
Question was asked by Furong Yan, while he was trying to use DAS tools, especially DAS client on coffea-casa.

**Comment by andrzejnovak** [comment]
fyi das client/certs can be installed in conda (https://hackmd.io/@andrzejnovak/SJxscCZvD) so it should be easy to add to the image even without cvmfs if a token can somewhat replace the x509 auth

**Reply by yanfr0818** [reply]
I'm having an issue when running `go get github.com/dmwm/das2go` when following the steps to install `dasgoclient` through `go` in the link.

cms-jovyan@jupyter-furong-2eyan-40cern-2ech:~/dasgoclient$ go get github.com/dmwm/das2go
cgo: exec x86_64-conda-linux-gnu-cc: exec: "x86_64-conda-linux-gnu-cc": executable file not found in $PATH

**Comment by holzman** [comment]
Flagging @mapsacosta  - we have CVMFS (with CentOS 7) working in EAF images ...

**Comment by oshadura** [comment]
@yanfr0818 @andrzejnovak @holzman @mapsacosta  In dasgoclient repo we requested some time ago a support of tokens: https://github.com/dmwm/dasgoclient/issues/30

Here is a current status (we still can't use it, but I will need to retest): https://github.com/CoffeaTeam/coffea-casa/issues/96#issuecomment-824828429

**Comment by mapsacosta** [comment]
@yanfr0818 For a more Kubernetes-y type solution, [here](https://github.com/mapsacosta/cvmfs-nfs-kubernetes) is my code for deploying an NFS server pod serving CVMFS repositories. The exported repos can then be mounted into user pods as Kubernetes NFS volumes or simple NFS mounts

** Note that restorting to NFS CVMFS exports is discouraged by developers but it's one of the simplest ways to do this while trying to keep least privilege principles ([CSI](https://gitlab.cern.ch/cloud-infrastructure/cvmfs-csi) is another option but it requires more complex, DaemonSet deployments)

**Reply by bbockelm** [reply]
Here's another DaemonSet approach - could easily be used via kustomize:

https://github.com/opensciencegrid/osg-k8s-cvmfs
